docs/guidance/identifiers.md

```diff
@@ -20,9 +20,14 @@ There are different approaches to including project identifiers in contracting d
 
   This approach also enables the joining up of data on multiple contracting processes relating to a single infrastructure project.
 
+
```

Comment:
Remove extra newline.

Reply:
Only one new line is rendered in the built docs, without the extra new line there is no space between the end of the paragraph and the next bullet.

Reply:
Strange –  that's fine then!